selenium_tests/ab_tool/ab_base_test_case.py

```diff
@@ -0,0 +1,36 @@
+
+from selenium import webdriver
+from selenium_tests.base_page_object import BasePageObject
+from selenium_tests.ab_tool.page_objects.pin_page import LoginPage
+from selenium_tests.base_test_case import BaseSeleniumTestCase
```

Comment:
same comment about organizing imports applies here.  core libraries first, then third-party modules, then our stuff.  in this case, there's no core libraries, so we'd start with a section that has the `django.conf` and `selenium` imports, followed by a section with the `selenium_tests` imports.